Library/Homebrew/formulary.rb

```diff
@@ -226,6 +228,13 @@ def self.loader_for(ref)
       return AliasLoader.new(possible_alias)
     end
 
+    if newref = FORMULA_RENAMES[ref]
+      formula_with_that_oldname = core_path(newref)
+      if formula_with_that_oldname.file?
+        return FormulaLoader.new(newref, formula_with_that_oldname)
+      end
+    end
```

Comment:
No, I changed everything that used `canonical_name` and could cause problems, so I think everything is OK.